src/backend/routers/post.py

```diff
@@ -179,6 +184,25 @@ async def get_comments(
     )
 
 
+@router.post("/{post_id}/rating")
```

Comment:
NIT: `post_id` should be `id`  for consistency.
Small disciplines repeated with consistency everyday lead to great achievements gained slowly over time.

Reply:
Hmm, in some place it's `post_id` whereas in others  it's `id`. We should make it consistent throughout. An inconsistent mind is disorganised and undisciplined.

I prefer `post_id` over `id` btw.

Reply:
I just copied the function for create comment here lmao. I can change the rest of the functions to use `post_id` if you want.

Reply:
I want. I need.

Reply:
Resolved in [fdff730](https://github.com/idkbrowby/gamingrealm-backend/commit/fdff7300c23bee59889690e6d59e161f9c4c59b6)